docker_grocy/www.conf

```diff
@@ -0,0 +1,6 @@
+[global]
+daemonize = no
+
+[www]
+clear_env = no
```

Comment:
@tet3 This is one potential candidate.  I don't love this change, since it potentially enables a whole bunch of environment variables to pass from the host to the PHP application.

If we could find a way to selectively allow only `GROCY_`-prefixed variables (i.e. genuine application variables) to the application, that could be an improvement.  Or perhaps there should be a larger design re-think here.

Reply:
Writing down these thoughts in a more structured form, to approximate requirements:

- The app's (non-default?) configured environment values should be committed to source control in this repository, so that they're easily visible and modified
- Only environment variables relevant to the application should be made available to the container(s) from the host
- It should be possible for end-users of the Docker images to override any of the application's environment variables at runtime -- without rebuilding the image
- Sensible default values should be provided in released Docker images so that new users can get started with only a command or two